lib/alerts/check_timeline_manager.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class CheckTimelineManager
+  def initialize(course)
+    @course = course
+    create_alerts
+  end
+
+  def create_alerts
```

Comment:
This should only create one (unresolved) alert for a given course.

I think this can be a `resolvable` alert, such that if an admin marks it as resolved (meaning the Timeline has been fixed so that it's now okay and includes training modules) then a second alert could be created.

Some of the other alert manager files have examples you can follow for how to implement this.